src/modules/Resources/Files.ts

```diff
@@ -431,15 +466,16 @@ class Files extends TagoIOModule<GenericModuleParams> {
    * @example
    * If receive an error "Authorization Denied", check policy **File** / **Upload** in Access Management.
    * ```typescript
-   * const file = Buffer.from("file content");
-   * const result = await Resources.files.uploadFile(file, "/uploads/myfile.txt", {
+   * const fileBuffer = Buffer.from("file content");
+   * const fileBlob = new Blob([fileBuffer]);
```

Comment:
- This change can be reverted, but it's good for front end since there's no `Buffer` in browsers
- It might be breaking change if someone uses `uploadFile` with a `Buffer` directly

Reply:
This change will be reverted after discussing with the team